### DISCUSSION TITLE
Get Current User Session

### DISCUSSION THREAD

**Discussion by yigitali01** [serenity-is/Serenity#5843] [Help]
Hey,

I want to add get current User Session with using:   UserDefinition user = (UserDefinition)Authorization.UserDefinition;

But it didn't work. After I saw that it can now be done by calling a new function that called User.GetIdentifier(). but it is not shown exactly how to call the function. Is there a way for me to call the id of the current user?

Thanks

**Comment by VictorTomaili** [comment]
Search for `Authorization.UserDefinition` in discussions. you can find solution.
